cascade/calculator.py

```diff
@@ -96,3 +98,52 @@ def make_calculator(
                 timeout=timeout,
                 debug=debug,
                 **cp2k_opts)
+
+
+class EnsembleCalculator(Calculator): 
```

Comment:
Could you add a docstring explaining how to use this? E.g., why is it different than a regular calculator, what new outputs does it yield?

Reply:
Missing the docstring on one of the main classes introduced by the PR was a pretty big oversight :laughing: